examples/shader/Program.cs

```diff
@@ -13,10 +13,7 @@ protected Effect(string name)
             myName = name;
         }
 
-        public string Name
-        {
-            get { return myName; }
-        }
+        public string Name => myName;
```

Comment:
@LaurentGomila that work for you too? We were thinking of waiting for .NET Core 2.0 so VB.NET support isn't dropped, and also to wait for OpenTK to support 2.0.